po/ro.po

```diff
@@ -0,0 +1,703 @@
+msgid "Calculating conflicts..."
+msgstr "Calcularea conflictelor..."
+
+#: src/install.rs:1017
+msgid "Calculating inner conflicts..."
+msgstr "Calcularea conflictelor interioare..."
+
+#: src/install.rs:1029
+msgid "Inner conflicts found:"
+msgstr "Conflicte interioare depistate:"
+
+#: src/install.rs:1051
+msgid "Conflicts found:"
+msgstr "Conflicte depistate:"
+
+#: src/install.rs:1074
+msgid "Conflicting packages will have to be confirmed manually"
+msgstr "Pachetele care intră în conflict vor trebui confirmate manual"
+
+#: src/install.rs:1078
+msgid "can not install conflicting packages with --noconfirm"
+msgstr "nu s-au putut instala pachetele care intră în conflict cu --noconfirm"
+
+#: src/install.rs:1117
+msgid "Repo"
+msgstr "Repo"
+
+#: src/install.rs:1124
+msgid "Repo Make"
+msgstr "Repo Make"
+
+#: src/install.rs:1138
+msgid "Aur Make"
+msgstr "Aur Make"
+
+#: src/install.rs:1240
+msgid "fetching devel info..."
+msgstr "se obțin devel info..."
+
+#: src/install.rs:1320
+msgid "Signing packages..."
+msgstr "Se semnează pachetele..."
+
+#: src/install.rs:1398 src/install.rs:1403
+msgid "failed to download sources for '{}'"
+msgstr "nu s-au putut descărca sursele"
+
+#: src/install.rs:1408 src/install.rs:1447 src/install.rs:1455
+msgid "failed to build '{}'"
+msgstr "nu s-a putut compila '{}'"
+
+#: src/install.rs:1411
+msgid "{}: parsing pkg list..."
+msgstr "{}: procesare lista pkg..."
+
+#: src/install.rs:1473
+msgid "adding {} to the install list"
+msgstr "se adaugă {} la lista de instalare"
+
+#: src/install.rs:1532
+msgid "could not find package '{pkg}' in package list for '{base}'"
+msgstr "nu s-a putut găsi pachetul '{pkg}' in lista de pachete '{base}'"
+
+#: src/install.rs:1670
+msgid "can't find package name in packagelist: {}"
+msgstr "nu s-a putut găsi numele pachetului in packagelist: {}"
+
+#: src/install.rs:1743
+msgid "There are {} members in group"
+msgstr "Există {} membrii în grup"
+
+#: src/install.rs:1768
+msgid ""
+"\n"
+"\n"
+"Enter a selection (default=all): "
+msgstr ""
+"\n"
+"\n"
+"Introdu pachetele selectate (default=toate): "
+
+#: src/install.rs:1965
+msgid "{}-{} is up to date -- skipping install"
+msgstr "{}-{} este actualizat la ultima versiune -- se sare peste instalare"
+
+#: src/keys.rs:47
+msgid "keys need to be imported:)"
+msgstr "cheile trebuie importate:)"
+
+#: src/keys.rs:52
+msgid "     {key} wanted by: {base}"
+msgstr "     {key} dorită de: {base}"
+
+#: src/news.rs:57
+msgid "No Date "
+msgstr "Nicio Dată"
+
+#: src/news.rs:60
+msgid "No Title"
+msgstr "Niciun Titlu"
+
+#: src/news.rs:69
+msgid "no new news"
+msgstr "nicio știre nouă"
+
+#: src/query.rs:105
+msgid " [ignored]"
+msgstr " [ignorat]"
+
+#: src/repo.rs:164
+msgid "failed to get current exe"
+msgstr "nu s-a putut obține current exe"
+
+#: src/repo.rs:195
+msgid "syncing local databases..."
+msgstr "se sincronizează baza de date locală..."
+
+#: src/repo.rs:201
+msgid "  nothing to do"
+msgstr " nimic de făcut"
```

Comment:
Missing space

Reply:
Saw the typo, fixed now, thanks for pointing it out.